rocky/tests/conftest.py

```diff
@@ -158,14 +158,24 @@ def organization_b():
 @pytest.fixture
 def superuser(django_user_model):
     return create_user(
-        django_user_model, "superuser@openkat.nl", "SuperSuper123!!", "Superuser name", "default", superuser=True
+        django_user_model,
+        "superuser@openkat.nl",
+        "SuperSuper123!!",
+        "Superuser name",
+        "default",
+        superuser=True,
```

Comment:
I think there is something wrong with the editor settings used, because there is a lot of formatting noise in this PR which makes it hard to review. For example this file has 13 formatting only changes and one real change as far as I can see. It would also be nice if changes are reviewed before they are committed or pushed, because such a review would also spot these kind of changes.

It would be nice if all those formatting changes in these and a lot of other files could be reverted to reduce the noise and potential merge conflicts.

Reply:
precommit reformatted it this way, is there a difference in version?

Reply:
I have reverted back the linting as it was